torch/csrc/jit/passes/tensorexpr_fuser.h

```diff
@@ -11,8 +12,15 @@ struct Graph;
 // Run TensorExpressions-based fuser.
 TORCH_API void fuseTensorExprs(std::shared_ptr<Graph>& graph);
 
-// Register TensorExpressions-based fuser in custom passes.
-TORCH_API void registerTensorExprFuser();
+struct TORCH_API RegisterTensorExprFuser
+    : public PassManager<RegisterTensorExprFuser> {
+  static void registerPass() {
+    PassManager::registerPass(fuseTensorExprs);
+  }
+  static void clearPass() {
+    PassManager::clearPass();
+  }
+};
```

Comment:
Provided everything works, the changes in `torch/csrc/jit/passes/tensorexpr_fuser.*` look good to me (I'll also reply in #34769).